security/advancedtls/advancedtls.go

```diff
@@ -322,8 +322,8 @@ func buildVerifyFunc(c *advancedTLSCreds,
 	}
 }
 
-// NewClient uses ClientOptions to construct a TransportCredentials based on TLS.
-func NewClient(o *ClientOptions) (credentials.TransportCredentials, error) {
+// NewClientCreds uses ClientOptions to construct a TransportCredentials based on TLS.
+func NewClientCreds(o *ClientOptions) (credentials.TransportCredentials, error) {
```

Comment:
Why adding surfix `Creds`?
This changes from `advancedtls.NewClient()` to `advancedtls.NewClientCreds()`. I think `NewClient()` was good.